packages/expo-in-app-purchases/src/ExpoModuleTemplateView.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+import { requireNativeViewManager } from '@unimodules/core';
+
+export default class ExpoInAppPurchasesView extends React.Component {
```

Comment:
Removable

Reply:
Left the View stuff in there because I was thinking about potentially implementing a View for this module, however it likely would've been Android only and may not have even gotten used much, so I'll remove it instead.